lib/Model/Listing/AbstractListing.php

```diff
@@ -411,8 +411,14 @@ public function setGroupBy($groupBy, $qoute = true)
         if ($groupBy) {
             $this->groupBy = $groupBy;
 
-            if ($qoute && strpos($groupBy, '`') !== 0) {
-                $this->groupBy = '`' . $this->groupBy . '`';
+          if ($qoute && !str_starts_with($groupBy, '`')) {
+                $quotedParts = [];
+                $parts = explode(",", $groupBy);
```

Comment:
```suggestion
          if ($qoute) {
                $quotedParts = [];
                $parts = explode(",", str_replace('`', '', $groupBy));
```
I mean, quoting should be done in any case, no matter if `$groupBy` starts with a  \` character or not.

Reply:
Aaah ok, sorry, now I get what you mean. Yes, that would be even better. I will do the same for setOrderKey then.

Reply:
```suggestion
          if ($qoute) {
                $quotedParts = [];
                $parts = explode(",", trim($groupBy, '`'));
```
Hmm .. don´t want to be petty, but technically it is possible to create columns with a backtick in the name. If we do a str_replace, then we can´t order by these columns anymore. 
Maybe trimming the backticks would be better, since all other backticks will be escaped by `quoteIdentifier`?

Reply:
Valid point 😊 LGTM 👍